src/core/Class.js

```diff
@@ -86,7 +86,14 @@ Class.extend = function (props) {
 // @function include(properties: Object): this
 // [Includes a mixin](#class-includes) into the current class.
 Class.include = function (props) {
+	var options;
+	if (props.options) {
+		options = Util.extend({}, this.prototype.options, props.options);

```

Comment:
Hi,

Why not simply leverage [`mergeOptions`](https://github.com/Leaflet/Leaflet/blob/v1.3.1/src/core/Class.js#L93-L98) to handle `include({options})` special case?
That would avoid breaking the reference with the parent class' options.
Example: https://plnkr.co/edit/Eth0KHPhwETsXUQwD9fq?p=preview

Reply:
BTW should we also treat as special cases [`statics`](https://github.com/Leaflet/Leaflet/blob/v1.3.1/src/core/Class.js#L43-L47) and… [`includes`](https://github.com/Leaflet/Leaflet/blob/v1.3.1/src/core/Class.js#L49-L54)?

Reply:
Yeah, that sounds good. My changes were rather a quick&dirty fix, but I certainly agree with fixing `statics` and `includes` too. Will need to write unit tests for those.

Reply:
Why do we need to introduce some unobvious 'special cases' where we can achieve the same in clear explicit way?